src/nuspec/docfx.console/build/docfx.console.targets

```diff
@@ -8,41 +8,43 @@
     <MetadataOutputFolder Condition=" '$(MetadataOutputFolder)' == '' ">$(OutputFolder)</MetadataOutputFolder>
     <LogFile Condition=" '$(LogFile)' == '' ">log.txt</LogFile>
     <LogLevel Condition=" '$(LogLevel)' == '' ">Verbose</LogLevel>
+    <BuildDocFx Condition=" '$(LogLevel)' == '' ">true</BuildDocFx>
```

Comment:
`BuildDocFx` => `BuildDocfx`, `LogLevel` => `BuildDocfx`. I think you can swap this line with below, so that it will always respect the value intentionally set.